internal/locate/region_request.go

```diff
@@ -113,7 +113,8 @@ type RegionRequestSender struct {
 	replicaSelector   ReplicaSelector
 	failStoreIDs      map[uint64]struct{}
 	failProxyStoreIDs map[uint64]struct{}
-	RegionRequestRuntimeStats
+	Stats             *RegionRequestRuntimeStats
+	AccessStats       *ReplicaAccessStats
```

Comment:
Should the `replicaAccessStats` be part of `RegionRequestRuntimeStats`?
From the current code path:
1. SendReqToRegion
2. RegionRequest
2.1 RPC request
2.2 Replica selection
2.3 Retry and error handling

Reply:
`ReplicaAccessStats` is a little different with `RegionRequestRuntimeStats` , so put `ReplicaAccessStats` in `RegionRequestRuntimeStats` will be a little inappropriate.

- `ReplicaAccessStats` will record all replica access patch which meets error in current send-req round, but `RegionRequestRuntimeStats` will collect all rpc count and duration by type.
  - And collect all `ReplicaAccessStats` for a SQL will be too expensive.
- `ReplicaAccessStats` currentlly only used when print region request slow log, but `RegionRequestRuntimeStats` will be used in plan executor runtime stats.

Reply:
Can we some graph to show the relashionships bettween them from the persipective of `SQL latency analysis`?
For example the SQL execution phases are:
```
SQL ---> To KV Requests ---> LoadRegion ---> SendReqToRegion ---> BuildRPC Context -> Select Replia -> ..
                                                                                    <---                        retry
                                                     <---                                                       retry                                                         
```
So we need to know how the `ReplicaAccessStats` and `RegionRequestRuntimeStats` could help to analyze the latatency in the above phases.

Reply:
great, done. PTAL